### DISCUSSION TITLE
A similar "flux identity" command in flux v2

### DISCUSSION THREAD

**Discussion by dimakyriakov** [fluxcd/flux2#2903] [Q&A] [answered]
We have our own Golang application that uses old flux and there we use "flux identity" command. Currently, we want to go with fluxV2 and looking for the similar command in the new version of flux. Can somebody help?

**Comment by stefanprodan** [comment, marked as answer]
How are you installing Flux? Please see the [bootstrap docs](https://fluxcd.io/docs/installation/#github-and-github-enterprise), there is no need for an identity command, `flux bootstrap github` sets the deploy key for you in GitHub.
